README.md

```diff
@@ -1,8 +1,6 @@
-[Website](https://www.bitcoinunlimited.info)  | [Download](https://www.bitcoinunlimited.info/download) | [Setup](doc/README.md)  |  [Xthin](doc/bu-xthin.md)  |  [Xpedited](doc/bu-xpedited-forwarding.md)  |   [Miner](doc/miner.md)
-
```

Comment:
why remove this header?  It makes a nice quick-access bar to common documentation.

Reply:
I removed it also because I deleted doc/README.md
But even if you look at the current doc/README.md it is a mix of many different info, and really isn't much help if you want to "setup" your node.

My overall thought was to split the documentation up into:

- Installing (from binaries)
- Configuration of a node
- Building (from source)
- Contributing to the source

Having both a /README.md and a doc/README.md seemed confusing to me.
So I ended up having /README.md as a sort of "index" of all the other files in /doc

If you think another way of organizing the documentation is better, I would be happy to hear it.

Adding the header back is also ok with me, I'm just not sure it helps people navigate the docs.

Reply:
@soeren-b-c I agree with your approach, but still I think @gandrewstone has a point. The "quick-access" bar give an easy way to people to access to essential documents and also have a function of showing what are they key techs behind Bu (xthin, xpedited etc etc).

Reply:
Will add it (or something like it) back the next couple of days.